src/Vault.sol

```diff
@@ -122,10 +122,12 @@ contract Vault is IVault, VaultToken, Ownable {
     }
 
     function sync(Currency currency) public override isLocked {
-        VaultReserve.alreadySettledLastSync();
-        if (currency.isNative()) return;
-        uint256 balance = currency.balanceOfSelf();
-        VaultReserve.setVaultReserve(currency, balance);
+        if (currency.isNative()) {
+            VaultReserve.setVaultReserve(CurrencyLibrary.NATIVE, 0);
```

Comment:
i wonder if we can have a helper function so its clearer eg. `VaultReserve.clearVaultReserve()` or something

Reply:
if agreeable, we can create an issue to tackle in another PR (so it doesn't pollute this)

Reply:
https://github.com/pancakeswap/pancake-v4-core/issues/166

Reply:
😎hello